src/entity/units/golem.ts

```diff
@@ -14,14 +14,14 @@ const unit: UnitSource = {
     subtype: UnitSubType.MELEE,
   },
   unitProps: {
-    damage: 30,
-    staminaMax: config.UNIT_BASE_STAMINA,
-    healthMax: 20,
+    damage: 20,
```

Comment:
Same damage as the archer?

Reply:
Yes. Do you feel that's too low/high?

The basic golems are meant to be a little weaker in this version